runtime-sdk/src/types/transaction.rs

```diff
@@ -175,7 +175,8 @@ pub struct Fee {
     /// Maximum amount of gas paid for.
     #[cbor(optional)]
     pub gas: u64,
-    /// Maximum amount of emitted consensus messages paid for.
+    /// Maximum amount of emitted consensus messages paid for. Zero means that up to the maximum
```

Comment:
I think using zero makes it easier to use because this field is already deployed and just omitting the field will result in the new behavior be in effect (which makes the most sense for most use cases). There is no reason to actually have this field now, and we can consider removing it in a later iteration.